dist/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8" />
+        <title>Codidact - Helping each other learn.</title>
+        <meta
+            name="description"
+            content="Codidact is an open-source, community-controlled Q&amp;A platform. Currently under active development. Contribute by joining our team."
+        />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+        <link rel="manifest" href="site.webmanifest" />
+        <link rel="apple-touch-icon" href="icon.png" />
+
+        <link rel="stylesheet" href="assets/css/normalize.css" />
+        <link rel="stylesheet" href="assets/css/primary.css" />
+    </head>
+
+    <body>
+        <!--[if IE]>
+            <p class="browserupgrade">
+                You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to
+                improve your experience and security.
+            </p>
+        <![endif]-->
+
+        <div class="container-centered">
+            <p class="heading-big">Codidact</p>
+            <p class="subtitle">
+                A community-controlled, open-source Q&amp;A platform.
+            </p>
+            <p class="subtitle">Work in progress.</p>
+            <p class="heading-small">Contribute</p>
+            <p class="subtitle">
```

Comment:
Indentation in `<p>` tags results in funny blank space when copying text from the page ;)

(I'm fixing this on my working copy)

Reply:
Fixed at 21c6195c